colefiscus/whatsCookinRefactor
# Feature/favorites display refactor

### Is this a fix or a feature?
Fix.

### What is the change/fix?
Made sure stars show up to indicate favorite, and persist when changing pages. Update show favorites to only display recipes in the user's favorites.

### Where should the reviewer start?
viewFavorites function, favoriteCard and associated helper methods. getFavorites.

### How should this be tested?
Run the application and open in localhost:8080 and see if everything is functioning properly.

### Screenshots(if appropriate)
